### PR TITLE
Added browser check bar on top of the window [v.2.6.0]

### PR DESCRIPTION
**Pull request by aklochkova**
Fixes https://github.com/OHDSI/Atlas/issues/1091

### CODE REVIEW

File: js/Model.js [hunk 1]
```diff
@@ -53,6 +55,9 @@ define(
 			constructor() {
 				super();
 				const bemHelper = new BemHelper('app');
+				const browser = detect.parse(navigator.userAgent);
+				this.isBrowserSupported = browser.browser.family.toLowerCase() === 'chrome' && parseInt(browser.browser.version) > 63;
```
Comment (pavgra) [inline js/Model.js]: Move detection of whether browser is supported to utils

File: js/app.less [hunk 2]
```diff
@@ -7,6 +7,22 @@
     bottom: 0;
     background-color: #003142;
     border-right: solid 1px #888;
+
+    &--with-warning {
```
Comment (pavgra) [inline js/app.less]: Why do you need this? Why cannot the warning bar just move things down?

File: js/settings.js [hunk 3]
```diff
@@ -130,6 +130,7 @@ const settingsObject = {
     "colorbrewer": "../node_modules/colorbrewer/index",
     "ohdsi-api": "../node_modules/@ohdsi/ui-toolbox/lib/umd/api/index",
     "bootstrap-datetimepicker": "../node_modules/bootstrap-datetimepicker/src/js/bootstrap-datetimepicker",
+    "detect-browser": "https://cdnjs.cloudflare.com/ajax/libs/Detect.js/2.2.2/detect",
```
Comment (pavgra) [inline js/settings.js]: use NPM

File: js/Model.js [hunk 4]
```diff
@@ -53,6 +53,7 @@ define(
 			constructor() {
 				super();
 				const bemHelper = new BemHelper('app');
+				this.isBrowserWarningShown = ko.observable(!commonUtils.isBrowserSupported);
```
Comment (pavgra) [inline js/Model.js]: `isBrowserSupported` seems to be function:
`const isBrowserSupported = function () {`

File: js/styles/browserWarning.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.browser-alarm {
+    background-color: #ce8483;
+    color: darkred;
+    height: 30px;
+    display: flex;
+    align-items: center;
+}
+
+.browser-alarm .warning-text {
```
Comment (johnSamilin) [inline js/styles/browserWarning.css]: Please follow BEM
`.browser-alarm`
`.browser-alarm__warning-text`
`.browser-alarm__close`

File: js/utils/BrowserDetection.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+
+const browser = detect.parse(navigator.userAgent);
+const isBrowserSupported = browser.browser.family.toLowerCase() === 'chrome' && parseInt(browser.browser.version) > 63;
+if (!isBrowserSupported){
+    document.getElementById("b-alarm").style.visibility = "visible";
+} else {
+    hideWarning();
+}
+
+function hideWarning() {
```
Comment (johnSamilin) [inline js/utils/BrowserDetection.js]: `toggleWarning` maybe?

File: js/utils/CommonUtils.js [hunk 7]
```diff
@@ -188,6 +188,7 @@ define([
 		createConceptSetItem,
 		syntaxHighlight,
 		getPathwaysUrl,
-		normalizeUrl
+		normalizeUrl, 
+		isBrowserSupported
```
Comment (pavgra) [inline js/utils/CommonUtils.js]: What do you export here? Where is the function definition?

File: js/utils/CommonUtils.js [hunk 8]
```diff
@@ -129,7 +129,7 @@ define([
 		conceptSetItem.includeMapped = ko.observable(false);
 		return conceptSetItem;
 	}
-
+
```
Comment (pavgra) [inline js/utils/CommonUtils.js]: Dummy diff

File: js/utils/CommonUtils.js [hunk 9]
```diff
@@ -2,7 +2,7 @@ define([
 		'knockout',
 		'atlas-state',
 		'pages/Page',
-		'urijs'
+		'urijs',
```
Comment (pavgra) [inline js/utils/CommonUtils.js]: Dummy diff

File: js/assets/detectBrowser.js [hunk 10]
```diff
@@ -0,0 +1,1196 @@
+/**
```
Comment (pavgra) [inline js/assets/detectBrowser.js]: Use NPM. Include as a separate `<script>` tag or do whatever, but with NPM

**Comment by pavgra** [comment]
@aklochkova ,

1. We use NPM to manage dependencies. **NPM**, not locally stored files
1. Have you looked at repo of the library which you have chosen? It is no longer maintained
image

**Pull request merged**